src/frontend/src/components/SearchPage/SearchResults.jsx

```diff
@@ -54,7 +57,18 @@ const SearchResults = ({ text, filter }) => {
 
   return (
     <Container className={classes.searchResults}>
-      {data && data.length ? <Timeline pages={[data]} /> : <h1>No results search</h1>}
+      {data && data.length ? (
+        <Timeline
+          pages={data.map((d) => d.values)}
```

Comment:
Can we do this map when we get the results vs. here?  That is, let's have `data` be in the format we expect so it doesn't have to be altered.

Reply:
Also, `pages` expects an array of arrays.

Reply:
yeap, I had data take both the results, and values property from the fetch to try and write logic for last page detecting but I'll change it back since we'll move that logic to the backend